server/api/include/rsExecRuleExpression.hpp

```diff
@@ -3,7 +3,6 @@
 
 #include "exec_rule_expression.h"
 
-int rsExecRuleExpression(rsComm_t*,exec_rule_expression_t*);
+int rsExecRuleExpression(rsComm_t* comm,exec_rule_expression_t* input);
```

Comment:
space after comma

Reply:
don't know how i missed that.  it has been updated.